src/components/FacetSelect.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2018, The Linux Foundation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import Select from 'react-select'
+
+import 'react-select/dist/react-select.css';
+
+const options = [
+  { value: 'core', label: 'Core'},
+  { value: 'data', label: 'Data'},
+  { value: 'dev', label: 'Dev'},
+  { value: 'docs', label: 'Docs'},
+  { value: 'examples', label: 'Examples'},
+  { value: 'tests', label: 'Tests'}
+]
```

Comment:
I don't know if we want this in here or in the PageComponent to generalize this further.

Reply:
Feels fine here... it could be another prop but at least for now, Facets and the list are pretty hard concepts in the system. We'll see if we need this same sort of control and want generalize it to just a "multiselector" or something.